Bento/Renderable/Renderable.swift

```diff
@@ -33,4 +33,19 @@ public extension Renderable {
             didEndDisplayingItem: didEndDisplayingItem
         ).asAnyRenderable()
     }
+
+    func customInput(
+        _ input: CustomInput,
+        contentStatus: FocusEligibility.ContentStatus = .empty,
+        highlightColor: UIColor? = UIColor(red: 239/255.0, green: 239/255.0, blue: 244/255.0, alpha: 1),
+        backgroundColor: UIColor = .white
```

Comment:
Why do we need a background color? We are just wrapping it into a decoration view which can have a clear, right?

Reply:
Forgot that I invented `HighlightingGesture.stylingView`.